flask_server.py

```diff
@@ -49,36 +70,49 @@ def index():
     rendered_base64_media = []
     cookie_path = settings["cookie_path"] if "cookie_path" in settings else "token.txt"
     folder_path = settings["folder_path"]  if "folder_path" in settings else "ig_media"
-
     if request.method == "POST":
         amountScraped = int(request.form["amountToScrape"])
         mode = request.form["mode_dropdown"]
         count_i, count_v, base64_media = start_scrape(cookie_path, folder_path, amountScraped, mode)
         rendered_base64_media = render_base64_media(base64_media)
         log_line = set_log_file_list()
     return render_template('index.html', count_i = count_i, count_v = count_v, log_line = log_line, images = rendered_base64_media)
-
 	
 @app.route("/settings/", methods=['GET','POST'])
 def settings():
-
     settings = get_settings()    # Gets the settings
     cookie_path = settings["cookie_path"] if "cookie_path" in settings else "token.txt"
     folder_path = settings["folder_path"]  if "folder_path" in settings else "ig_media"
-
+    updated_settings = {}
     if request.method == "POST":
-        updated_settings = {}
-
         for setting in request.form:
             if len(request.form[setting]) > 0:
                 updated_settings[setting] = request.form[setting]
         save_settings(updated_settings)
+    return render_template("settings.html", folder_path = updated_settings["folder_path"] if "folder_path" in updated_settings else folder_path, cookie_path = updated_settings["cookie_path"] if "cookie_path" in updated_settings else cookie_path)
```

Comment:
Move the cookie_path = , folder_path, just above this lien and use them in the function,